package-lock.json

```diff
@@ -0,0 +1,10660 @@
+{
```

Comment:
FYI we have a yarn.lock file: https://github.com/all-contributors/all-contributors-bot/blob/master/yarn.lock

Reply:
Ah yeah, I'll remove that (I'm too used to `npm`).